modules/standard/Regexp.chpl

```diff
@@ -42,9 +42,9 @@ in the Chapel distribution.
 
 .. note::
 
-  if re2 support is not enabled (which is the default), all features
-  described below will compile successfully, but will result in an internal
-  error at *run time*, saying "No Regexp Support".
+  if re2 support is not enabled (which is the case in quickstart
```

Comment:
Could possibly cross-reference `:ref:quickstart-with-other-shells` to make it clear what 'quickstart' is.